formats/gltf/writer.go

```diff
@@ -496,11 +496,43 @@ func (w *Writer) AddMesh(model PolyformModel) (_ int, err error) {
 }
 
 func (w *Writer) AddTexture(polyTex PolyformTexture) *TextureInfo {
```

Comment:
I think this is a valid spot to re-use the same logic we did with Meshes where we keep up with the pointers of the Textures and check the addresses instead of the values of the textures themselves. Thoughts?

Reply:
Yep, that works as well.